src/core/prism.ts

```diff
@@ -48,7 +48,7 @@ export class Prism {
 		this.hooks.run('before-highlightall', env);
 
 		assertEnv<'before-all-elements-highlight'>(env);
-		env.elements = [...env.root.querySelectorAll(env.selector)];
+		env.elements = Array.from(env.root.querySelectorAll(env.selector));
```

Comment:
Is there a particular reason why `Array.from()` is better than the spread operator? 🤔

Reply:
Good catch, I had this as a holdover from another attempt.